src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -399,7 +399,7 @@ def __call__(
                 callback(i, t, latents)
 
         latents = 1 / 0.18215 * latents
-        image = self.vae.decode(latents).sample
+        image = torch.cat([self.vae.decode(latent).sample for latent in latents.split(1)])
```

Comment:
Could we please add a `enable_vae_slicing()` for this? I don't think all the backends such as MPS like the "for loop". Happy to add this features with a `enable_vae_slicing`

Reply:
this list comprehension is iterating over a regular python array, not indexing into a tensor. I don't think MPS would have trouble with that. a similar pattern is working for me on MPS (split a tensor then cat the splits):  
https://github.com/apple/ml-ane-transformers/blob/da64000fa56cc85b0859bc17cb16a3d753b8304a/ane_transformers/reference/multihead_attention.py#L116

Reply:
Thanks! Added enable_vae_slicing() in the latest commit.